moulinette/interfaces/api.py

```diff
@@ -387,6 +387,15 @@ def process(self, _route, arguments={}):
             ret = self.actionsmap.process(arguments, timeout=30, route=_route)
         except MoulinetteError as e:
             raise error_to_response(e)
+        except Exception as e:
+            if isinstance(e, HTTPResponse):
+                raise e
+            import traceback
```

Comment:
Yea, I dunno what's the best practice about that... On one hand it's cleaner to import it at the beginning, on the other hand it's a module only used in edge cases so it's useless to load it in 99.9% of cases... Personnaly I don't have a strong opinion about this, I just implemted this that way because we have this in other part of the code...